Documentation/upgrade.md

```diff
@@ -410,7 +410,7 @@ kubectl -n rook get pod -l app=rook-ceph-mds -o jsonpath='{range .items[*]}{.met
 At this point, your Rook cluster should be fully upgraded to running version `rook/ceph:v0.8.0` and the cluster should be healthy according to the steps in the [health verification section](#health-verification).
 
 ## Upgrading Kubernetes
```

Comment:
You could actually remove this whole section. We haven't supported TPRs for a couple releases.